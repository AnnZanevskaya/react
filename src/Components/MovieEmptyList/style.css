.container{
    text-align: center;
    height: 400px;
    display: flex;
    justify-content: center;
    flex-direction: column;
}

.description{
    color: #fff;
    font-size: 2.2rem;
    font-weight: 400;
}