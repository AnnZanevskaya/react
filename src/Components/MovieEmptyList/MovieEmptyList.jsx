import React from "react";

import './style.css';

const MovieEmptyList = () => (
    <div className="container">
        <p className="description">No Movie Found</p>
    </div>
);

export default MovieEmptyList;
