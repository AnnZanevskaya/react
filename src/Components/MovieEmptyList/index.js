import MovieEmptyList from './MovieEmptyList';

export default MovieEmptyList;
