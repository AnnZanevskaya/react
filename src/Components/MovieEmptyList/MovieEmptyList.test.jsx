import React from 'react';
import renderer from 'react-test-renderer';
import MovieEmptyList from './MovieEmptyList';

it('renders correctly', () => {
  const tree = renderer
    .create(<MovieEmptyList />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});