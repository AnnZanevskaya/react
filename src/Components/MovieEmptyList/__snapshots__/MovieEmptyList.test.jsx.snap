// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container"
>
  <p
    className="description"
  >
    No Movie Found
  </p>
</div>
`;
